accession/utils.py

```diff
@@ -0,0 +1,123 @@
+"""Returns configurations for exporting querysets to csv based on the model.
+
+This module has one function (get_csv_config) which takes a model and returns
+a list and two dictionaries. The list (fields) dictates which fields are
+to be returned in a queryset from the given model. The first dictionary
+(serializer_map) maps new fields to their data, and is used here to return
+the string representation of a foreign key in the Accession model. The second
+dictionary (header_map) renames field names. It is used to make the final csv
+output more easily readable. The list and dictionaries are determined by what
+model is given.
+"""
```

Comment:
For instance, this is the _utils_ module, so any new utility could be added to, rendering the statement `This module has one function...` incorrect. It is not a big deal to update in the future, but in my experience docstrings and module docs are often overlooked when making changes.